modules/payloads/singles/windows/meterpreter_reverse_https.rb

```diff
@@ -57,6 +58,7 @@ def generate
       Rex::Payloads::Meterpreter::Patch.patch_passive_service! dll,
```

Comment:
I'll go through and fix them up. That stuff was older code, but now's definitely the time to fix 'er up.